sklearn/feature_selection/tests/test_from_model.py

```diff
@@ -101,6 +185,10 @@ def test_feature_importances():
     mask = np.abs(transformer.estimator_.coef_) > 1e-5
     assert_array_equal(X_new, X[:, mask])
```

Comment:
Oh, I see. `threshold=None` used to mean "default threshold". Now you want it to mean no threshold (i.e. `threshold=-np.inf`). It needs to continue meaning "default threshold" for good or bad. You should document that a user may use `threshold=-np.inf` if they only want `max_n_features` to apply.